src/command/upload.ts

```diff
@@ -5,8 +5,9 @@ import * as Path from 'path'
 import { sleep } from '../utils'
 import { Tag } from '@ethersphere/bee-js/dist/types'
 import { SingleBar, Presets } from 'cli-progress'
-import { bold, green, dim, red } from 'kleur'
+import { bold, green, dim } from 'kleur'
```

Comment:
These should probably not be used here, right?
```suggestion
```

Reply:
It uses "bold" and "green" still, the commit suggestion is wrong